tests/integration/async_push_test_client.py

```diff
@@ -236,7 +237,8 @@ async def send_notification(
         # Pull the sent notification immediately if connected.
         # Calls `get_notification` to get response from websocket.
         if self.ws and self.ws.is_client:  # check back on this after integration
-            return object.__getattribute__(self, "get_notification")(timeout)
+            res = await object.__getattribute__(self, "get_notification")(timeout)
+            return res
```

Comment:
Good call @pjenvey . Relatedly, was it in convo with you or @jrconlin that it was suggested the `"get_notification"` calls be made explicitly instead of in this 'clever' way, since I know when I first saw it and looked at the resultant return values in the tests that it was a bit baffling at first.